src/main/java/Duke.java

```diff
@@ -79,81 +81,96 @@ public static CommandStatus handleListCommand() {
                 entryNum++;
             }
         }
+
         printLineSeparator();
 
         return CommandStatus.SUCCESS;
     }
 
 
     public static CommandStatus handleDoneCommand(String commands[]) {
+        if (commands.length < 2){
+            return CommandStatus.FAIL;
+        }
         String taskDescription = commands[1];
 
         // Looks for the matching description task to mark as done
         boolean taskFinished = false;
```

Comment:
For example here taskFinished =  true;
I think you should use isFinished for boolean variables.